src/main/g8/src/main/scala/Main.scala

```diff
@@ -1,4 +1 @@
-object Main {
-  def main(args: Array[String]): Unit =
-    println("Hello, world!")
-}
+@main def hello = println("Hello world from Scala Native!")
```

Comment:
I, for one, __like__ this change and would like to see it proceed.

Not an earth shaking changes, but I have "Hello world", of various
flavors, popping into my log files all over the place.  Always nice to
know who is speaking, just like radio protocol.

Reply:
I think there was `none` `lto` because it was failing on the Mac. As long as we have a NativeConfig so the user doesn't have to figure out that ceremony I am good.